spinners.json

```diff
@@ -1278,5 +1278,25 @@
 			"βββββρβ",
 			"ββββββρ"
 		]
+	},
+		"fish": {
+		"interval": 150,
+		"frames": [
+			">))'>",
+			"<'((<"
```

Comment:
I don't think this is clear enough as being as fish when animated. To me, it just looks like noise. I would drop this one.

Reply:
okayy